docs/content/developer/getting-started/simple-token-transfer.mdx

```diff
@@ -0,0 +1,436 @@
+---
+description: Tutorial for simple token transfer
+tags:
+  - explanation
+  - tutorial
+---
+
+# Simple Token Transfer Tutorial
+
+In this tutorial, we will walk you through the process of building a simple token transfer dApp using the IOTA Move language and the IOTA dApp kit. You will learn how to create a contract that enables token transfers on the IOTA blockchain and how to integrate it with a frontend application that interacts with the contract's functions. The tutorial covers everything from contract creation to frontend development, giving you a hands-on understanding of how token transfers work within the IOTA ecosystem. By the end, you'll have a fully functional dApp that allows users to transfer tokens effortlessly.
+
+## Simple Token Transfer Architecture Overview 
+
+```mermaid
+sequenceDiagram
+    participant Sender
+    participant Contract
+    participant Treasury
+    participant Recipient
+
+    Sender->>Contract: init(witness, ctx)
+    Contract->>Treasury: create_currency(witness)
+    Treasury-->>Contract: Returns TreasuryCap, Metadata
+    Contract->>Sender: transfer(treasury, sender)
+
+    Sender->>Contract: mint(treasury_cap, amount, recipient, ctx)
+    Contract->>Treasury: mint(treasury_cap, amount)
+    Treasury-->>Contract: Returns Coin
+    Contract->>Recipient: transfer(coin, recipient)
+
+    Sender->>Contract: transfer(token, amount, recipient, ctx)
+    Contract->>Treasury: split(token, amount)
+    Treasury-->>Contract: Returns CoinToTransfer
+    Contract->>Recipient: transfer(CoinToTransfer, recipient)
+
+    Sender->>Contract: balances(token)
+    Contract-->>Sender: Returns token.balance
+```
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org/) >=v20.18.0
+- [npx](https://www.npmjs.com/package/npx) >=10.8.2
+- [iota](https://github.com/iotaledger/iota/releases) >=0.7.3 
+
+## Create a Move Package
+
+Run the following command to create a Move package
+
+```bash
+iota move new token-transfer
```

Comment:
`Invalid package name. Package name must start with a lowercase letter and consist only of lowercase letters, numbers, and underscores.`
```suggestion
iota move new token_transfer
```